### PR TITLE
fixed a typo in neo4j

### PR DESCRIPTION
**Pull request by supperpiccle**
Hey Dr. Stefano, I *fixed* the typo on betweennessCentrality.  Look over it to make sure it is right (sum might be a keyword I am not aware of.)

### CODE REVIEW

File: csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala [hunk 1]
```diff
@@ -257,12 +257,12 @@ object Neo4jWorkload extends Workload {
...
-    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*..$k]->(b)) " +
-      "WHERE n IN nodes(pthroughn) AND n <> a AND n <> b AND a <> b " +
-      "WITH n,a,b,count(pthroughn) AS sum " +
-      s"MATCH p = shortestPath((a)-[*..$k]->(b)) " +
-      "WITH n, a, b, tofloat(sumn)/ tofloat(count(p)) AS fraction " +
-      "RETURN n, sum(fraction);"
+    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*]->(b))" +
+    "WHERE n IN nodes(pthroughn) AND n <> a AND n <> b AND a <> b" +
+    "WITH n,a,b,count(pthroughn) AS sumn" +
+    "MATCH p = shortestPath((a)-[*]->(b))" +
```
Comment (scordio) [inline csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala]: Same as the previous.

File: csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala [hunk 2]
```diff
@@ -257,12 +257,12 @@ object Neo4jWorkload extends Workload {
...
+    "WITH n, a, b, tofloat(sumn)/ tofloat(count(p)) AS fraction" +
+    "RETURN n.name, sum(fraction);"
```
Comment (scordio) [inline csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala]: Returning only the name property means that you are losing all the additional properties, and we don't want it.

File: csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala [hunk 3]
```diff
@@ -257,12 +257,12 @@ object Neo4jWorkload extends Workload {
    * @return Graph containing the betweenness double values
    */
   def betweennessCentrality[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED], k: Int): Unit = {
-    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*..$k]->(b)) " +
-      "WHERE n IN nodes(pthroughn) AND n <> a AND n <> b AND a <> b " +
-      "WITH n,a,b,count(pthroughn) AS sum " +
-      s"MATCH p = shortestPath((a)-[*..$k]->(b)) " +
-      "WITH n, a, b, tofloat(sumn)/ tofloat(count(p)) AS fraction " +
-      "RETURN n, sum(fraction);"
+    val query = s"MATCH (n), pthroughn = shortestPath((a)-[*]->(b))" +
```
Comment (scordio) [inline csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala]: Why did you replace `[*..$k]` with `[*]`? This way the iterations are not bounded.

**Review by scordio — CHANGES_REQUESTED** [review]
Take a look at these minor changes.

File: csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala (hunk 3 above)
Comment (supperpiccle) [reply csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala]: I copied and pasted what was sent to me in an email.  I'll try to do a combination of the two

**Review by scordio — APPROVED** [review]

**Pull request merged**